### PR TITLE
[PLGN-432] Dig Version Bump To 2.0.1

### PR DESCRIPTION
**Pull request by abmccann-r7**
### Description

Describe the proposed changes:

  - Added empty `__init__.py` file and refreshed with new tooling

- Note: the `CloudReadyValidator` is going to fail as the `apk add` has been kept in the Dockerfile and this is an os level dependency. Removing the `apk add` causes issues and a ticket has been raised in regards to the investigation as to how to remove this line and keep the plugin functioning.

- Added new package to requirements.txt

- Implemented unit tests

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``rapid7/insightconnect-python-3-38-slim-plugin:{sdk-version-num}`` and ``rapid7/insightconnect-python-3-38-plugin:{sdk-version-num}``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``insight-plugin validate`` which calls ``icon_validate`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `insight-plugin samples`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `insight-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `insight-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `insight-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `insight-plugin run --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `insight-plugin validate` and make sure everything passes
2. Run the assessment tool: `insight-plugin run -A`. For single action validation: `insight-plugin run tests/{file}.json -A`
3. Copy (`insight-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

File: plugins/dig/help.md [hunk 1]
```diff
@@ -1,49 +1,40 @@
...
-
-* A
-* AAAA
-* ANY
-* CNAME
-* MX
-* NS
-* PTR
-* SOA
-
```
Comment (cmcnally-r7) [inline plugins/dig/help.md:40]: Is it okay to have this removed now? Not sure what it means & its very uncommon to see but lmk

**Review by cmcnally-r7 — APPROVED** [review]
Just one small thing but other than that lgtm 😄

File: plugins/dig/help.md (hunk 1 above)
Comment (abmccann-r7) [reply plugins/dig/help.md:40]: Is it the format that looks odd or the content? It's just the types of queries you can use in the required input

Comment (rbowden-r7) [reply plugins/dig/help.md:40]: yeah I would keep this, these represent the DNS query type for the dig command, eg

`dig example.com A` will return the ipv4 of exampe.com
`dig example.com AAAA` will return the ipv6 of exampe.com
`dig example.com MX` will return the mail exchange of exampe.com

and so on so these examples are worth keeping in

Comment (rbowden-r7) [reply plugins/dig/help.md:40]: As these the required fields that the user can enter are provided in the example below as part of the enum field this looks ok that it was removed

image

**Review by rbowden-r7 — APPROVED** [review]

**Pull request merged**